easybuild/easyconfigs/b/binutils/binutils-2.26-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'binutils'
+version = '2.26'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
+description = "binutils: GNU binary utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+builddependencies = [
+    ('flex', '2.6.3'),
+    ('Bison', '3.0.4'),
+    # zlib required, but being linked in statically, so not a runtime dep
+    ('zlib', '1.2.11'),
+    # use same binutils version that was used when building GCC toolchain, to 'bootstrap' this binutils
+    ('binutils', version, '', True)
+]
+
+patches = [
+    'binutils-%(version)s_build-with-GCCcore-6.3.0.patch',
+]
```

Comment:
Please collapse this to a single line, and move this up right below `sources` (and also include a checksum for the patch, `eb --inject-checksums` will do this for you).